Drop Python 2.7 (#936)
python-attrs/attrs: changelog.d/936.breaking.rst

Python 2.7 is not supported anymore.

> The tooling around Python 2.7 has become too tedious for a volunteer-run project.

Reviewer: ```suggestion
Dealing with Python 2.7 tooling has become too difficult for a volunteer-run project.
```
